src/features/game/events/landExpansion/feedChicken.ts

```diff
@@ -21,26 +26,30 @@ type Options = {
 const makeFedAt = (
   inventory: Inventory,
   collectibles: Collectibles,
-  createdAt: number
+  createdAt: number,
+  bumpkin: Bumpkin
 ) => {
-  if (
-    inventory["Wrangler"]?.gt(0) &&
-    isCollectibleBuilt("Speed Chicken", collectibles)
-  ) {
-    return (
-      createdAt - CHICKEN_TIME_TO_EGG * (0.1 + MUTANT_CHICKEN_BOOST_AMOUNT)
-    );
-  }
-
+  const { skills } = bumpkin;
+  const decimalCreatedAt = new Decimal(createdAt);
```

Comment:
Why are we using a Decimal for time here?

Reply:
> Why are we using a Decimal for time here?
To avoid  imprecision when doing the math